.ci.yaml

```diff
@@ -4161,13 +4186,17 @@ targets:
         }
     bringup: true
 
-  - name: Mac_ios native_assets_ios_simulator
+  - name: Mac native_assets_ios_simulator
```

Comment:
Is it intended to run this target on either x64 or arm? Others are updated to `Mac_x64`.

Reply:
Yes, I was a PR reviewer when this one got added so I have context on it. It should be okay to run on either

Reply:
I assume this PR will not affect any benchmark metrics? Just double confirm.

Reply:
I'm unsure how to check that. Do you know how?

Reply:
Benchmark metrics are test by test, some are focusing on host side whereas others focus on devices. I am not sure what these tests focus on though. 
@zanderso Do you have any idea how to tell? Or test owners may have a better idea.

Reply:
FYI: I think I figured out how to check. I'm comparing the times from the tests I ran in presubmit (https://github.com/flutter/flutter/pull/136416/checks?check_run_id=17647037514) to the graph in the dashboard.

Example:
https://flutter-flutter-perf.skia.org/e/?queries=test%3Dmacos_chrome_dev_mode&selected=commit%3D37338%26name%3D%252Carch%253Darm%252Cbranch%253Dmaster%252Cconfig%253Ddefault%252Cdevice_type%253DiPhone_11%252Cdevice_version%253Dnone%252Chost_type%253Dmac%252Csub_result%253DInitialStartupTime%252Ctest%253Dmacos_chrome_dev_mode%252C